docker-compose.prod.yml

```diff
@@ -85,9 +83,7 @@ services:
       - rabbitmq
       - minio
   client:
-    build:
-      context: .
-      dockerfile: ./dev/client.Dockerfile
```

Comment:
same here, leave this in until we have it properly building the containers.  There is more configuration at the github settings level needed before we have publicly available containers.

Reply:
Re-added in 86616ee